protocols/kad/src/behaviour/test.rs

```diff
@@ -73,7 +73,7 @@ fn build_node_with_config(cfg: Config) -> (Multiaddr, TestSwarm) {
         behaviour,
         local_id,
         swarm::Config::with_async_std_executor()
-            .with_idle_connection_timeout(Duration::from_secs(5)),
+            .with_idle_connection_timeout(Duration::from_secs(10)),
```

Comment:
We shouldn't need to increase this. Which tests fail without this?

Reply:
None of them, I changed this because I found that origin default `connection_idle_timeout` duration was 10 secs in PR #4659. Should we keep origin setting or set it shorter?

Reply:
Can we get away with not setting an idle timeout at all? That would be even better!

Reply:
I think there are some tests that are based on a default timeout. https://github.com/libp2p/rust-libp2p/pull/4659#issuecomment-1764636057

Reply:
Okay thanks!

Can you reduce it down to 5 again then?

Reply:
changed 3e75728